### PR TITLE
chore(criterion-output-item): add criterion peer review output item

### PR DESCRIPTION
**Pull request by sijav**
this can be merge AFTER #201 pull request

### CODE REVIEW

File: src/shared/evaluation-output/EvaluationOutput.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import { EvaluationOutput } from './EvaluationOutput';
+
+storiesOf('Evaluation Output', module)
+  .addDecorator(themeDecorator())
+  .add('Self', () => {
+    return (
+      <Container>
+        <Grid container spacing={2}>
+          <Grid item xs={12}>
+            <EvaluationOutput type="self" value={Evaluation.EXCEEDS_EXPECTATIONS} />
```
Comment (c0m1t) [inline src/shared/evaluation-output/EvaluationOutput.stories.tsx]: add one for `value=null`

File: src/shared/criterion-output-item/CriterionPeerReviewOutputItem.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import { Styles } from 'src/shared/types/Styles';
+
+interface OwnProps {
+  evaluation: '%future added value' | Evaluation | null;
+  evidence: string | null;
+}
+
+type Props = FCProps<OwnProps> & StyleProps;
+
+export function CriterionPeerReviewOutputItem({ evaluation, evidence, ...otherProps }: Props) {
```
Comment (c0m1t) [inline src/shared/criterion-output-item/CriterionPeerReviewOutputItem.tsx]: ProjectPeerReveiwOutput

File: src/shared/evaluation-output/EvaluationOutput.tsx [hunk 3]
```diff
@@ -10,14 +10,14 @@ interface OwnProps {
   type: 'self' | 'peer';
 }
 
-type Props = FCProps<OwnProps>;
+type Props = FCProps<OwnProps> & TypographyProps<'span', {}>;
```
[the remark this answers is not in the document]
Comment (sijav) [reply src/shared/evaluation-output/EvaluationOutput.tsx]: Removed from this changeset

File: src/shared/evaluation-output/EvaluationOutput.stories.tsx (hunk 1 above)
Comment (sijav) [reply src/shared/evaluation-output/EvaluationOutput.stories.tsx]: Removed from this changeset

File: src/shared/criterion-output-item/CriterionPeerReviewOutputItem.tsx (hunk 2 above)
Comment (sijav) [reply src/shared/criterion-output-item/CriterionPeerReviewOutputItem.tsx]: Done

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import { Styles } from 'src/shared/types/Styles';
+
+interface OwnProps {
+  evaluation: '%future added value' | Evaluation | null;
+  evidence: string | null;
+}
+
+type Props = FCProps<OwnProps> & StyleProps;
+
+export function ProjectPeerReviewOutput({ evaluation, evidence, ...otherProps }: Props) {
```
Comment (c0m1t) [inline src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]:
```suggestion
export function ProjectPeerReviewOutput(props: Props) {
const {evaluation, evidence} = props;
```

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+  const classes = useStyles(otherProps);
```
Comment (c0m1t) [inline src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]:
```suggestion
  const classes = useStyles(props);
```

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+          </Typography>
+          <MultilineOutput value={evidence || NON_BREAKING_SPACE} className={classes.outputTypography} />
+        </Grid>
+      </Grid>
+    </Box>
+  );
+}
+
+const styles = (theme: Theme) => ({
+  outputBox: {
```
Comment (c0m1t) [inline src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: root

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    backgroundColor: '#f5f5f5',
+    borderRadius: 4,
+    padding: theme.spacing(2, 1.5),
+  } as CSSProperties,
+  outputTypography: {
```
Comment (c0m1t) [inline src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: typography

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+    color: '#000',
```
Comment (c0m1t) [inline src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: `theme.palette.common.black`

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx (hunk 4 above)
Comment (sijav) [reply src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: Will open another PR to fix these thanks

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx (hunk 5 above)
Comment (sijav) [reply src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: Will open another PR to fix these thanks

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx (hunk 6 above)
Comment (sijav) [reply src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: Will open another PR to fix these thanks

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx (hunk 7 above)
Comment (sijav) [reply src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: Will open another PR to fix these thanks

File: src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx (hunk 8 above)
Comment (sijav) [reply src/shared/project-peer-reveiw-output/ProjectPeerReviewOutput.tsx]: Will open another PR to fix these thanks

**Pull request merged**